### ISSUE TITLE
colors masterpost

### ISSUE THREAD

**Issue by Th3Scribble** [lipu-linku/pali-nimi#23]
goals for color words:
1. list a bunch of english equivalents that fit the space (no semicolons)
2. have the most "central" colors be first
3. emphasize that walo and pimeja aren't just pure white and black
4. explain the colors in a way that it's clear that laso isn't "two colors" while all the others are just one
5. have overlap in the definitions where applicable

loje, jelo, laso, walo, and pimeja are the main words this issue is for. we will also discuss the "bright" aspect of suno's meaning

**Comment by Th3Scribble** [comment]
my proposal:
> loje: red, pink, orange, magenta, maroon
> jelo: yellow, golden, cream, lime, orange
> laso: teal, green, blue, cyan, violet, turquoise, sky, azure
> walo: pale, white, cream, pink, gray, unsaturated
> pimeja: dark, black, brown, maroon, navy blue, muddy color, gray
> suno: bright, vibrant, shiny {+ other defs}

jelo for orange and lime might be controversial. some of these translations may not be clear that they're colors and not the things themselves (like "sky") but i'd like to get feedback on that

**Comment by lipamanka** [comment]
i really like basically all of this for the reasons you mentioned.

**Comment by gtbot2007** [comment]
my ideas for the pu colors:

loje: red, pink, magenta, scarlet, vermilion (maybe orange if you want that to let be in two places)
jelo: yellow, orange, lime
laso: cyan, green, blue, teal, indigo (notably _not_ the word purple/violet since some purples are clearly loje)
walo: white, pale, light gray
pimeja: black, dark brown, dark gray

and also:
unu: purple
kapesi: brown, gray, bage dull

**Comment by AcipenserSturio** [comment]
The transition from "central word, anything in its surroundings" to "central word, a list of examples of all its surroundings" is messing with my brain and I wanted to complain - but I guess this puts it *more* in line with how other definitions already work. I don't love it but I will :rocket: it

**Comment by Th3Scribble** [comment]
i'm not sure what you mean
i tried to start with a central word, then its surroundings, then more specific shades
though i just noticed i tried to do this with loje by starting with "red, pink" and thinking people would include magenta in their idea of pink, but then i included magenta later
maybe i should start with "red, magenta"
idk the order probably doesn't matter _too_ much

**Comment by RetSamys** [comment]
I think my complaint would be more that they seem a bit long, just a bit. I'd prefer 4 words max, per colour. Idk, it's arbitrary I know

**Comment by lipamanka** [comment]
@RetSamys I can't think of a good reason to limit it such, and because any technical limit would be much larger than four words, I think having a larger number makes more sense. but we can make it have a cutoff at some point. unless you can articulate the reason that you prefer four words maximum. I think that eight might be a good compromise? 

ALSO re: orange in two places. I am HEAVILY in favor of demonstrating semantic overlap by having the same english word be in the definitions of two or more toki pona words. I think we should do this *as much as possible* to show that different words can mean the same thing, though I'm willing to compromise if people feel strongly against that.

**Comment by wasokeli** [comment]
can someone post the Linku and pu definitions?

**Comment by janPensa** [comment]
pu definitions and current Linku definitions are exactly the same:

> **loje** - ADJECTIVE red, reddish
> **jelo** - ADJECTIVE yellow, yellowish
> **laso** - ADJECTIVE blue, green
> **walo** - ADJECTIVE white, whitish; light-coloured, pale
> **pimeja** - ADJECTIVE black, dark, unlit
> *(suno - ADJECTIVE sun; light, brightness, glow, radiance, shine; light source)*

**Comment by janPensa** [comment]
I spent some time writing my thoughts about the color words and the discussion so far. It got a bit long, so I added some headers for structure.

## shorter miscellaneous remarks

#### "sky"
> some of these translations may not be clear that they're colors and not the things themselves (like "sky") but i'd like to get feedback on that

For things like "orange" and "cream", I think in context it's clear enough that colors are meant and not the physical objects/subtances. But yeah, for "sky" I'd change it to "sky blue" or "sky-colored".

#### "unsaturated"

> walo: (...) unsaturated

This works relatively well in HSV color space, but it does not fit as well with HSL color space, or with how most artists use the term "saturation". Which, by the way, is why a lot of digital artists prefer HSL, and color pickers like this:

<img width="150" alt="color-triangle" src="https://github.com/lipu-linku/pali-nimi/assets/92953245/46d430a2-2cd6-4985-94d1-3f0e12daeb8b">

(In HSL, saturation corresponds to the X axis of this triangle. Note that #F00 goes towards #808080 when unsaturated, not towards #000.)

#### definition length

> [the definitions] seem a bit long, just a bit. I'd prefer 4 words max, per colour.

I think 4 is too few, especially for laso, where we need more examples than four to be clear that we mean the entire spectrum from green to cyan to navy blue (and perhaps indigo or something).

I'd aim at around 5-6 words, and have 8 or so as the maximum.

#### amber

I think "amber" is a very good one to add to the "jelo" definition. Personally, I'd put it as the second word, after "yellow". Amber is around the middle point between orange and the very bright primary color version of yellow, or you could say that it's a shade of yellow at the outer edge of the "orange" part of the spectrum

## on laso's "central" color

I think the first word in the definition for "laso" should probably be cyan, turquoise, teal, or a similar word.

Of those, three, I think turquoise is the best, for several reasons.

- if you look up examples of turquoise, both the color and the gem, you get a range of colors that includes ones I'd call "blue" *and* ones I'd call "green", not just ambiguous grue colors.
- "turquoise" seems to be the most well-known of the three. According to Google's Ngram viewer ["turquoise" is about 4 times more common than "cyan" and "teal"](https://books.google.com/ngrams/graph?content=turquoise%2Cteal%2Ccyan&year_start=1800&year_end=2019&corpus=en-2019&smoothing=3). Of course partially because of the gem, but I think the books in this corpus probably reference the color more often than the gem.
- Thanks to the gem, "turquoise" and its literal translations are a much more international name (including in many Germanic languages, Romance languages, a number of Slavic languages, and various non-Indo-European languages like Turkish, Arabic and Finnish). Non-native English speakers using the dictionary are going to find it easier to understand than e.g. "teal", and many Linku translators are going to have an easier time translating it.
- "teal" seems to be used a lot for dark shades of cyan/turquoise, so it's closer to the edge between laso and pimeja
- in many people's minds, "cyan" is specifically a color very similar to the primary color used in CMY(K) color schemes (hex code: #00FFFF), excluding too unsaturated variants or surrounding hues. Also, most shades of cyan are considered a type of blue by most English speakers, which is less the case with turquoise and teal.

## about including "orange" (and "gray") in two places

> I am HEAVILY in favor of demonstrating semantic overlap by having the same english word be in the definitions of two or more toki pona words.

I'm skeptical. I think this strategy only works well if you can count on people looking up both "loje" and "jelo" and comparing the definitions, or noticing the shared word. But with more casual use of the dictionary, I think more beginners are going to see one definition and make conclusions like "oh, I guess that orange = loje" without noticing that orange is listed under jelo as well. Listing the same word under two definitions and hoping people make the connection only really works well if you put the definitions side by side, and in Linku if you want to get the definitions of two related words side by side, you generally have to really go out of your way to find a good search term for that.

But, counterpoint: if beginners stay in the Toki Pona community for long enough, or if they use the Linku dictionary often enough, they are going to eventually notice that orange can be both loje and jelo. In the long term, it probably wouldn't matter much. But I still have the feeling that slapping "orange" into both definitions without explanations or clarifications could lead to unnecessary confusion among beginners. I might be wrong, though.

Also, the semantic space of the English word "orange" is so broad, that there are shades of orange that I'd personally never call "jelo", *as well as* shades of orange that I'd never call "loje". ***However***, if we add a qualifier, such as "reddish orange" at loje and "yellowish orange" at jelo, then I'd be totally in favor.

Similarly, I wouldn't put "gray" in both "walo" and "pimeja" definitions, but I do think it's a good idea to have "light gray" and "dark gray" in there.

**Comment by lipamanka** [comment]
I think "yellowish orange" and "reddish orange" is a good compromise, but I can't think of any actual harm that could come from someone only noticing orange under either jelo or loje but not both. like, orange fits into both word's semantic spaces. a beginner doesn't need to know about the overlap directly, because words are presented individually. but cutting a piece out of a semantic space *because* of said overlap is kind of weird because it doesn't show the semantic space of one of the words to its full extent. do you see what I'm saying? I'm kind of rambling. I can give other examples if you don't understand. 

The only reason I'm hesitant to just go ahead and put "yellowish orange" under jelo is because I don't want to have to do this for other words too. I think like the goals here is `a definition accurately describes the semantic space of its word as accurately as possible` and overlap shouldn't matter. but we have biases because of these overlaps, so in order to correct for those intentionally adding in overlap in multiple words is good and we should do it.

**Comment by janPensa** [comment]
I guess that my problem with "orange" is less so that it is at the overlap between loje and jelo, and more that I don't feel like "orange" overlaps enough with either loje or jelo. I.e. "orange" includes too many colors that aren't loje *and* too many colors that aren't jelo.

![image](https://github.com/lipu-linku/pali-nimi/assets/92953245/5d50ae26-e38d-4d2b-8319-ccd425a6cf61)

In this example image, I'd never call any color on the bottom row "jelo", and I'd never call any on the top row "jelo" (nor the one they labled "orange" in this image, first from the left in the middle row). But almost all of them I'd call orange without hesitation (except for the ones I marked with a blue X, which are too red/yellow/beige for me to call them orange)

If all (or the vast majority of) shades of orange were both jelo and loje, I wouldn't mind as much if we put it in both definitions. But "orange" is too broad. That's why splitting its color range up into "yellowish orange" and "reddish orange" seems useful to me, if we want to mention oranges at all.

And now that I think of it, once again the same holds true for "gray". There are many colors that are definitely "gray" but not "walo", and many that are definitely "gray" but not "pimeja".

**Comment by lipamanka** [comment]
I think this gets to the point where we're thinking too much about *western color theory.* I don't think people look at "orange" and then assume it includes 
<img width="16" alt="image" src="https://github.com/lipu-linku/pali-nimi/assets/111026347/6dc4ec74-c356-4988-a175-05a612b2a0ec">
or 
<img width="15" alt="image" src="https://github.com/lipu-linku/pali-nimi/assets/111026347/e28b3e12-4b36-4c02-bea4-93834de2bda9">
right? they're gonna think of 
<img width="219" alt="image" src="https://github.com/lipu-linku/pali-nimi/assets/111026347/b3ac0187-4ad4-4e51-9880-9c834a4a5c4e">
all of which could probably be "jelo" or "loje."

**Comment by janPensa** [comment]
Well, that's very subjective. For me personally, the left part of 
<img alt="image" width="219" src="https://user-images.githubusercontent.com/111026347/262177357-b3ac0187-4ad4-4e51-9880-9c834a4a5c4e.png">
seems too jelo to call it loje.

Also, in my experience, the first color people think of when they hear "orange" can vary a lot. For me, I the prototypical orange is more like  
![image](https://github.com/lipu-linku/pali-nimi/assets/92953245/6a384db6-0447-4624-b091-67d1eda4ddab)
but when learning about color theory in various art classes, I've seen plenty of red-yellow-blue-based color wheels where this more yellowy color was labeled as orange (which, to be fair, is much closer to the color of the fruit):
![image](https://github.com/lipu-linku/pali-nimi/assets/92953245/0807cbdb-217b-458d-b4ca-bbe9ffec12a9)
and on those diagrams the "orange" I knew from all the markers and crayons of my youth was instead marked as "red-orange" or something like that.

Hearing "orange", some people will first think of these colors:
![image](https://github.com/lipu-linku/pali-nimi/assets/92953245/6f2206a8-00a3-44ee-bd6c-4b47d5edceb8)
and others think first of these:
![image](https://github.com/lipu-linku/pali-nimi/assets/92953245/74a4ebe4-5ecc-4c62-be78-1e5bb89cb5cd)

Those two ranges include these colors, of which I think there is going to be little disagreement about whether they are more jelo or loje.
![image](https://github.com/lipu-linku/pali-nimi/assets/92953245/9cdcf499-966d-4277-b374-64d86affae10)

Here they are next to "pure" yellow and red, for comparison
![image](https://github.com/lipu-linku/pali-nimi/assets/92953245/4abe3867-715e-44d2-9b71-2f2630a711ff)


**TL;DR:** the word "orange" on its own can mean too many different things to various people, and it definitely includes colors that are not very loje or not very jelo. (Especially if you then also consider that users of Linku are going to be from very different cultures, which can each have a different average opinion about what a typical example of the color orange is.)

Therefore, I think it's not very useful to include "orange", unless we use "yellowish/reddish orange" or something similar

**Comment by lipamanka** [comment]
why don't we just add nalanja to the (this joke is bad) okay fine

**Comment by janPensa** [comment]
Here's a concrete proposal for definitions that I'd be content with 🚀-ing. All are 5-6 translations, including various "core" examples, followed by one or two closer to the edges of the ranges.

> **loje** - red, magenta, scarlet, pink, rust-colored, reddish orange
> **jelo** - yellow, amber, golden, lime yellow, yellowish orange
> **laso** - turquoise, blue, green, cyan, indigo, lime green
> **walo** - light-colored, white, pale, light gray, cream
> **pimeja** - dark-colored, black, dark gray; unlit, shadowy, darkness

We could also perhaps add "reddish purple" and "bluish violet" or something like that. But I'm not sure about that.

<hr />

As for non-pu color words...

At the current "unu - purple" I'd definitely add "violet". And adding "beige" to kapesi, as @gtbot2007 suggested, sounds like it might be a good idea, but maybe we should try to ask some kapesi users first.

Definitions for "nalanja" and "peta" seem fine as they are.

**Comment by gtbot2007** [comment]
I am a kapesi user lol

**Comment by gtbot2007** [comment]
also I use jelo for lime green not laso

**Comment by janPensa** [comment]
> I am a kapesi user lol

Oh, good! :) But still, more than one person would be nice.

> also I use jelo for lime green not laso

lime colors seem to be the edge between laso and jelo for most people. I thought that using "lime yellow" and "lime green" would be a good way to distinguish the more jelo and more laso shades, but maybe we should change or remove those.

Still, I'd *prefer* to have an example of a more limey/fluorescenty shade of yellow included, because other than the neutral "yellow", all other examples imply a more orangey shade. Perhaps "tennis ball yellow", lol.

**Comment by wasokeli** [comment]
- i like yellowish-orange and reddish-orange
- similarly, reddish-purple and bluish-purple might be good for laso and loje
- i think it would be helpful for pimeja to include specific examples of dark non-grayscale colors, like brown and purple
- i personally use "pimeja" to mean "night". i'm not sure how common that is

**Comment by lipamanka** [comment]
> i like yellowish-orange and reddish-orange

big agree
> similarly, reddish-purple and bluish-purple might be good for laso and loje

big agree
> i think it would be helpful for pimeja to include specific examples of dark non-grayscale colors, like brown and purple

brown and purple can both be pretty light - can we think of better colors like "charcoal" and "coffee?" though those are also nouns. hmm I will try to find better examples via https://en.wikipedia.org/wiki/List_of_colors:_A%E2%80%93F
> i personally use "pimeja" to mean "night". i'm not sure how common that is

this is something I keep forgetting to bring up. pimeja isn't *just* a color word, in the same way suno isn't either. pimeja refers to shadows, dark things, etc. which includes the night. I know in pu "of a year" is listed under sike, which definitely follows in modern usage. Perhaps we can say "of the night" under pimeja? Altogether I would like to rework pimeja's definition to highlight and underscore its usage in contexts that aren't about colors. I will think of specifics here and perhaps draft a pimeja thing at some point.

**Comment by lipamanka** [comment]
possible colors to put in pimeja (though a lot of them are obscure so idk) 
<img width="139" alt="image" src="https://github.com/lipu-linku/pali-nimi/assets/111026347/ca016531-cbf1-482c-994b-79b850b2c2b1">
<img width="144" alt="image" src="https://github.com/lipu-linku/pali-nimi/assets/111026347/0edce9b4-83b1-4117-ab37-725e8e768b08">
<img width="114" alt="image" src="https://github.com/lipu-linku/pali-nimi/assets/111026347/22f7c972-4eb0-4239-b771-34cf9e971c54">

**Comment by gtbot2007** [comment]
sienna could work but not violet

**Comment by lipamanka** [comment]
hey, gtbot, just so we know: who are you? how did you find out about the project?

**Comment by janPensa** [comment]
> > i like yellowish-orange and reddish-orange
> 
> big agree
> 
> > similarly, reddish-purple and bluish-purple might be good for laso and loje
> 
> big agree

Yeah, now you two say so, I'm convinced that that's probably a better option than giving examples near the edge (like "lime").

What about the jelo-laso border, though?

Going by the same pattern, we should go with "greenish lime" and "yellowish lime", but that seems weird. Perhaps "greenish yellow" under "jelo" and "yellowish green" under "laso" could work? idk.

On the other hand, we could also just not include anything to define the border, because to a lot of people the jelo/laso border is already almost identical to the yellow/green border. If any lime-adjacent color is more yellow than green to me, I say "jelo" (in my case, a tennis ball or [an Australian fire truck](https://commons.wikimedia.org/wiki/File:ACTFB_tankers.jpg), for example), and if it's more green than yellow to me, I'd call it "laso" (e.g. [this iguana](https://commons.wikimedia.org/wiki/File:Brachylophus_fasciatus_Fidschileguan.jpg))

Thoughts?

<hr/>

> i think it would be helpful for pimeja to include specific examples of dark non-grayscale colors, like brown and purple

I searched for a good word for that, but the best I found was "taupe". "Onyx" and "ebony" are, in common parlance, just basically black or dark gray. Going with "dark purple" or "midnight blue" or some other adjective + any color of the rainbow seems like a bad idea to me. Like, encroaching too much on the loje/jelo/laso territory.

I guess a solution could be to use "black, or any (very) dark color" and "white, or any (very) light or pale color"

Any thoughts and ideas?

<hr/>

> i personally use "pimeja" to mean "night". i'm not sure how common that is

> Perhaps we can say "of the night" under pimeja? I would like to rework pimeja's definition to highlight and underscore its usage in contexts that aren't about colors

Yeah, that's why I included "unlit, shadowy, darkness" in my draft for pimeja. (Switching to a noun at "darkness" because "dark" is also color-related.)

"unlit" is in pu, and I thought that it is useful. Not super satisfied about "shadowy". IIRC I considered including something with "night" because of "tenpo pimeja" but I couldn't think of something specific that I thought works good, and I figured that "tenpo pimeja" works fine as "unlit time" or "dark time".

Oh. No, wait.

I'm probably confusing it with the description I wrote for the sona.pona.la article Hecko made a few days back, before I was aware of the discussion here. On there I wrote:

> *pimeja* is also used for "darkness", as an antonym of *suno* ("light"), and can be used to describe shadows, unlit spaces, and the night.

Personally, I wouldn't say that pimeja = night, though. pimeja is darkness, and night is one common form of darkness, but putting just "night" in the definition seems a bit strange to me. But not so much that I can't be convinced, if enough of you think differently.

**Comment by lipamanka** [comment]
I use pimeja by itself to mean night, because the night can be a type of darkness. I think this is reflected somewhat in common usage. But putting "night" seemed like it would be less descriptive of usage, that's why I suggested mirroring pu's sike def "of a year" by saying "of the night" because it also reflects common usage better.

**Comment by janPensa** [comment]
Oh, right after posting my last comment I realized: my hesitance with "night" probably comes from the fact that in my internal model of Toki Pona, any word that means "[noun]" can also mean "[noun]-related" when used as a modifier.

The idea that pimeja can mean "darkness-related" vibes as correct to me (although I can't think of good examples), but I wouldn't use pimeja for night-related things. Nocturnal birds aren't "waso pimeja" unless they're dark-colored, and night activities aren't "pali pimeja".

> that's why I suggested mirroring pu's sike def "of a year" by saying "of the night" because it also reflects common usage better.

Oh, I didn't notice the similarity to "of a year". Interesting.

But yeah, that makes sense. I'm not opposed if more people like this idea.

**Comment by lipamanka** [comment]
> Nocturnal birds aren't "waso pimeja" unless they're dark-colored, and night activities aren't "pali pimeja".

huh. both of these make perfect sense to me. but like if you are not opposed if more people like it then yay! Maybe this would help: why can't activities done in the dark, i.e. night, be darkness-related activities? Why can't birds that are only awake in the darkness, i.e. night, darkness-related birds? For so much of human history the night has been a type of darkness in so many contexts.

**Comment by gregdan3** [comment]
o lukin e ni @janPensa o


"o lukin a! waso pimeja li lon!"

"aaa, seme? ona li pimeja ala. sina seme??"

"a! taso, ona li lape lon tenpo suno, li alasa li moku li pali lon tenpo pimeja. la ona li pimeja."

"aaaa sona"

jan tu li ken kepeken nasin ante li awen ken sona kute e ona ante

mi la nasin ni pi nimi pimeja li ken tan poka pi ijo pimeja

**Comment by lipamanka** [comment]
ehhhhhhh "ona li pimeja" idk if this is as true as ending with "ona li waso pimeja" because while as a modifier it can mean "related to dark things," as a verb it can't as easily mean that.

**Comment by gregdan3** [comment]
you're getting stuck on the English analysis I think 

sina sona ala pimeja e waso la o toki pona e waso pimeja, sama ni:

waso pimeja a.
ona li waso li pimeja.
ona li waso pimeja.
waso li pimeja.


ni tu tu li ken pana e sona sama. ante li ken. taso sina toki e ni, toki ni li pona: ona li waso pimeja 
taso toki ante mi li ken pana e sona sama tan ni: sina sona e waso ona

**Comment by gregdan3** [comment]
actually, let me offer another, maybe more useful way of looking at it:

there is no such thing as "true" in toki pona, no way to say with 100% certainty that a given word is right or wrong. there are only words which are more or less appropriate at different times, appropriateness being measured by how well your listener understands you.

if I say "ona li pimeja" to refer to a they (bird) which is night related but not black, that probably wouldn't be understood out of the gate

but with the explanation of why I am using the word pimeja that way, it would be fine 

that's more my point than anything; prior explanations change understanding drastically, to the point of influencing somebody's nasin into the future in some cases (notable example: esun to refer to breathing)

**Comment by lipamanka** [comment]
pilin mi li ante. "waso pimeja" la "waso li pimeja" anu seme? tenpo la ni li ken. tenpo ante la ni li ken ala. waso li walo li lon pimeja taso la ona li pimeja ala. ona li waso pimeja li walo. anu seme. ni li pilin mi. taso nasin ante li ike ala suli tawa mi.

**Comment by gregdan3** [comment]
"tenpo la ni li ken. tenpo la ni li ken ala"
i don't think we're disagreeing lol

**Comment by lipamanka** [comment]
mi toki e ni: selo waso li pimeja la ni li ken. selo waso li walo li pimeja ala la ni li ken ala.

**Comment by gregdan3** [comment]
sina la ni li ken ala

mi toki e ken ni: waso li pimeja ala lon selo, taso ona li pali mute lon tenpo pimeja li lape tas o lon tenpo suno la ona li ken pimeja tan ni

ni li ken nasin nasa, taso sina sona e nasin la ken, anu seme

**Comment by lipamanka** [comment]
lon nasin nasa sina ni la ken. nasa ni li ike ala tawa mi. taso ona li nasa.

**Comment by wasokeli** [comment]
hmm i'm inclined to leave out "night" now

i understand "pimeja la" as an abbreviation for "tenpo pimeja la", but i wouldn't understand "waso pimeja" to mean "nocturnal bird"

**Comment by wasokeli** [comment]
like if "night" were the first word in the definition of pimeja, that would be misleading. having it as the last word isn't much better

**Comment by lipamanka** [comment]
well night is an example of darkness, so it does make it much better to me. we could add an "e.g." there if we need it but it's not misleading because we can put words that are more specific examples after the first one

**Comment by lipamanka** [comment]
once again i'm suggesting "of the night," not just "night"

**Comment by gregdan3** [comment]
in any case, while night and "of the night" both could be appropriate, they both are easily derived from dark/black/shadow etc in the same way below is derived from foot for noka

i don't think it's necessary to include either in the definition

i also don't think pimeja exhibits the "of one night" property as strongly as sike exhibits "of one year", even if that use is present; readers would probably be more likely to discover """"tenpo dropping"""" and apply this logic to more words than to say pimeja works this way and somehow it and sike are the only ones that do that

it feels out of place and likely to mislead 

the "e.g." variation for night doesn't seem bad tho

**Comment by lipamanka** [comment]
oh god tenpo dropping. what a weird overanalysis of suno semantically shifting to mean day on its own. there's no tenpo dropping. but i digress. i think we should also put "of one day" under suno and perhaps "consider "of one month" under mun. descriptively they are used like that very often so it wouldn't be just sike and pimeja, it would be the other ones that are commonly used like that. one could even argue that esun has begun to mean "of a week" though i'd personally be fine with leaving that one out.

**Comment by janPensa** [comment]
> in any case, while night and "of the night" both could be appropriate, they both are easily derived from dark/black/shadow etc in the same way below is derived from foot for noka
> 
> i don't think it's necessary to include either in the definition
 
This describes my feelings very well.

> one could even argue that esun has begun to mean "of a week"

That's actually not a new development. People have been using that since pre-pu times, back when lexicalization was rampant and some people felt a need for a set expression for "week". [This pre-pu corpus](https://github.com/mvuets/talk-processing-toki-pona-with-perl/tree/master/demo/corpus) includes a diary entry from 2009 that uses "tenpo esun pini la mi tawa ma tomo Okajoma."

But I digress.

> i think we should also put "of one day" under suno and perhaps "consider "of one month" under mun.

I'm learning more towards just putting "year", "month", and "day" in the definitions for "sike", "mun" and "suno", because they're used often enough on their own in those meanings. Perhaps with an asterisk that refers to a commentary explaining "tenpo sike" etc.

But it looks like we're leaning towards not including "night" or "of the night" (except perhaps in an e.g.), so unless other people join the discussion who are in favor of adding one of those, I think that this matter is mostly off-topic, and we probably shouldn't spend more than short comments on it until we're properly tackling sike, mun, and/or suno.

**Comment by lipamanka** [comment]
> Perhaps with an asterisk that refers to a commentary explaining "tenpo sike" etc.

i'm strongly against this because it's in essence encouraging lexicalized compound.

but in any case, have we considered saying "dark things, e.g. night, shadows" ? it follows the derivational patterns of adjective to noun.

**Comment by janPensa** [comment]
Perhaps then:

> **pimeja** - dark-colored, black; unlit, dark; something dark, e.g. darkness, shadows, night

Yeah, I think that works well. :)

And then mirroring it:

> **walo** - light-colored, white, pale

I think if we put "light/dark-colored" first, we don't really need extra examples to show that walo and pimeja aren't just "white" and "black". Which comes in handy, because pimeja's definition is already pretty crowded even without the 'e.g."

**Comment by wasokeli** [comment]
i think it would be helpful to give people a little nudge to consider pimeja for colors like neutral brown and neutral purple, which they currently don't do
which is why kapesi and unu have gotten so popular

**Comment by wasokeli** [comment]
dark color, e.g. black, purple, brown;

**Comment by wasokeli** [comment]
notably, natlangs with toki pona's color inventory use pimeja for those colors 
(as well as laso/loje for purple, and loje for brown)

**Comment by gtbot2007** [comment]
The english language concept of purple is definitely not pimeja. 
![A0E7559F-A6FC-4C20-A90D-0F577E6095AD](https://github.com/lipu-linku/pali-nimi/assets/78619370/76caf821-bff0-4346-abdc-4f98d67f842b)

Also while some browns could definitely be pimeja (if you don’t use the word kapesi, that is), most people would probably not use pimeja for beige or other light browns.

**Comment by wasokeli** [comment]
i tried to clarify this with "dark color, e.g. black, purple, brown"
but it's possible that there's another phrasing that clarifies that i mean neutral purple and neutral brown

**Comment by gtbot2007** [comment]
But neutral purple is definitely not pimeja.

**Comment by janPensa** [comment]
I'm sure that different people have very different ideas of what shade of purple is "neutral". Darker purples are more popular in clothing and products than lighter purples, but definitely not everyone is going to think about a dark color when they hear "purple" or "neutral purple".

Browns sort of tend to be dark, so "brown" *could* work. But there are also plenty of common brown colors that I'd not call pimeja. Like brown paper bags, and many wood colors. Those browns aren't represented in [that color words study](https://www.pnas.org/doi/10.1073/pnas.09109811) that you found recently, because they didn't test any desaturated colors, except for very light and very dark ones.

If we want to include purple and brown in the pimeja definition, I think it would have to be "dark purple" and "dark brown". But if we add color names with "dark" in front, I start feeling: why those and not "dark red", "dark blue", "dark pink", etc.?

And besides, it takes up space in a definition that's already likely to be quite long. So I'm leaning towards not adding any of those. Except maybe dark brown? idk
